packages/fxa-payments-server/src/routes/Product/index.scss

```diff
@@ -36,6 +37,7 @@
     img.avatar.hoisted {
       // TO DO: clean this up, see issue #3336
       margin-top: -$main-content-padding-top-desktop - 42px; // half of avatar height
+      z-index: 999;
```

Comment:
Do we need this if it's no longer sitting on top of anything else?

Reply:
Pushed an update to remove this.